src/attributes.rs

```diff
@@ -220,7 +220,7 @@ With endian-ness:
 ```rust
 # use core::convert::{TryInto, TryFrom};
 # use deku::prelude::*;
-# #[cfg(feature = "bits")]
+# #[cfg(all(feature = "alloc", feature = "bits"))]
```

Comment:
This shouldn't need alloc.

Reply:
My intent was to keep it consistent with the config guard on `main()`, but you're right, and I'll drop it.